From: Piotr Luc <piotr.luc@intel.com>
Subject: powercap / RAPL: Add Knights Mill CPUID
References: fate#321924
Patch-Mainline: v4.10-rc1
Git-commit: bd9089162d8002666f734dd18552c7297118fec3


Signed-off-by: Thomas Renninger <trenn@suse.de>

Add Knights Mill (KNM) to the list of CPUIDs supported by intel_rapl

Signed-off-by: Piotr Luc <piotr.luc@intel.com>
Reviewed-by: Dave Hansen <dave.hansen@intel.com>
Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

---
 drivers/powercap/intel_rapl.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/powercap/intel_rapl.c
+++ b/drivers/powercap/intel_rapl.c
@@ -1118,6 +1118,7 @@ static const struct x86_cpu_id rapl_ids[
 	RAPL_CPU(INTEL_FAM6_ATOM_DENVERTON,	rapl_defaults_core),
 
 	RAPL_CPU(INTEL_FAM6_XEON_PHI_KNL,	rapl_defaults_hsw_server),
+	RAPL_CPU(INTEL_FAM6_XEON_PHI_KNM,	rapl_defaults_hsw_server),
 	{}
 };
 MODULE_DEVICE_TABLE(x86cpu, rapl_ids);
